pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -131,6 +137,12 @@ protected void compareRowEquals(List<Object[]> resultRows, List<Object[]> expect
         return ((String) l).compareTo((String) r);
       } else if (l instanceof Boolean) {
         return ((Boolean) l).compareTo((Boolean) r);
+      } else if (l instanceof BigDecimal) {
+        if (r instanceof BigDecimal) {
+          return ((BigDecimal) l).compareTo((BigDecimal) r);
+        } else {
+          return ((BigDecimal) l).compareTo(new BigDecimal((String) r));
```

Comment:
is this b/c the return type of Pinot is not BigDecimal or b/c H2 is not returning BigDecimal?

Reply:
JSON represents them as strings - so in the cases of specified `outputs` we need to be able to compare them to strings.